opus/src/redux/reducer.js

```diff
@@ -2,10 +2,10 @@ const initialState = {
   token: null
 };
 
-const Reducer = (state = initialState, action) => {
+const reducer = (state = initialState, action) => {
```

Comment:
In case you implement other reducers later, give it a more explicit name - if this reducer is supposed to manage authentification, authReducer could be a possibility